### PR TITLE
fix(linux): `set_cursor_position` in window coordinates

### PR DESCRIPTION
**Pull request by lucasfernog**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: src/platform_impl/linux/window.rs [hunk 1]
```diff
@@ -563,15 +563,21 @@ impl Window {
...
 
-    if let Err(e) = self
-      .window_requests_tx
-      .send((self.window_id, WindowRequest::CursorPosition((x, y))))
-    {
+    if let Err(e) = self.window_requests_tx.send((
+      self.window_id,
+      WindowRequest::CursorPosition((
+        x + inner_pos.x,
+        y + inner_pos.y + outer_size.height as i32 - inner_size.height as i32,
```
Comment (amrbashir) [inline src/platform_impl/linux/window.rs]: This is enough, outer size and inner size is the same on Linux

```suggestion
        x + inner_pos.x,
        y + inner_pos.y,
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Comment by CaptainTux** [comment]
Should the same adjustment be done for the other platform implementations?
I think cursor repositioning is still in monitor coords on windows, isn't it?
Can test it at home in a few hours...

**Comment by amrbashir** [comment]
Nope, macOS and Windows are in Window cooridnates.

**Comment by CaptainTux** [comment]
Don't mind me then.
I just looked at the code and thought it wasn't 😕

**Pull request merged**